singularity-eos/eos/eos_davis.hpp

```diff
@@ -342,7 +368,7 @@ PORTABLE_INLINE_FUNCTION Real DavisReactants::Ps(const Real rho) const {
   }
 }
 PORTABLE_INLINE_FUNCTION Real DavisReactants::Es(const Real rho) const {
-  const Real y = 1 - _rho0 / rho;
+  const Real y = 1 - robust::ratio(_rho0, std::max(rho, 0.));
```

Comment:
Is this (max(rho,0)) desired?  Is it necessary to guard against negative rho?  If so, we should be consistent between above and here.

Reply:
Yes that is the intent. I'll correct the above